src/use-val.ts

```diff
@@ -1,6 +1,11 @@
 import { type ReadonlyVal } from "value-enhancer";
 
-import { useDebugValue, useEffect, useState } from "react";
+import { useCallback, useDebugValue } from "react";
+import { useSyncExternalStore } from "use-sync-external-store";
```

Comment:
Shouldn't it be `from 'use-sync-external-store/shim/index.js'`?

Reply:
Oh right, it should be `use-sync-external-store/shim`.